examples/server-side-form-validation-antd/src/pages/posts/edit.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { HttpError, IResourceComponentsProps } from "@refinedev/core";
+import { Edit, useForm, useSelect } from "@refinedev/antd";
+
+import { Form, Input, Select } from "antd";
+
+import MDEditor from "@uiw/react-md-editor";
+
+import { IPost, ICategory } from "../../interfaces";
+
+export const PostEdit: React.FC<IResourceComponentsProps> = () => {
+    const { formProps, saveButtonProps, queryResult } = useForm<
+        IPost,
+        HttpError,
+        IPost
+    >({
+        warnWhenUnsavedChanges: true,
+    });
```

Comment:
```suggestion
    >();
```
By default, it is already `true`